server/services/visitSessionsService.ts

```diff
@@ -52,7 +52,12 @@ export default class VisitSessionsService {
       })
     })
 
-    return { calendar, firstSessionDate }
+    // array of valid visitSession IDs (e.g. '2025-05-25_session-ref')
```

Comment:
As well as returning all calendar dates, times, etc also return just an array of the valid visit session IDs that can be stored in the session and will (next PR...) be used to validate that an allowable session has been selected.